tzdata improvements

filimonov opened the pull request:
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Expose version of embedded tzdata via TZDATA_VERSION in system.build_options.

Detailed description / Documentation draft:
Relates: https://github.com/ClickHouse/ClickHouse/pull/10425
Autogenerate list of shipped timezones. Script to check tzdb updates.

filimonov committed:
tzdata improvements

Expose version of embedded tzdata via TZDATA_VERSION in system.build_options.
Autogenerate list of shipped timezones. Script to check tzdb updates.

filimonov commented on contrib/cctz-cmake/CMakeLists.txt:
```diff
@@ -60,574 +60,17 @@ if (NOT EXTERNAL_CCTZ_LIBRARY_FOUND OR NOT EXTERNAL_CCTZ_LIBRARY_WORKS)
         set(TZ_OBJS)
 
+        # each file/symlink in that dir (except of tab and localtime) store the info about timezone
+        execute_process(COMMAND bash -c "cd ${TZDIR} && find * -type f,l -and ! -name '*.tab' -and ! -name 'localtime' | sort | paste -sd ';'" OUTPUT_VARIABLE TIMEZONES )
```
Not sure if it's good: from one side it's better to make it explicit, but from the other side it's too easy to forget update the list, and some timezones supported by tzdb wll not work. 

Right now we had those missing:
```
America/Nuuk
Etc/GMT+0
Etc/GMT-0
Etc/GMT0
Etc/GMT+1
Etc/GMT-1
Etc/GMT+10
Etc/GMT-10
Etc/GMT+11
Etc/GMT-11
Etc/GMT+12
Etc/GMT-12
Etc/GMT-13
Etc/GMT-14
Etc/GMT+2
Etc/GMT-2
Etc/GMT+3
Etc/GMT-3
Etc/GMT+4
Etc/GMT-4
Etc/GMT+5
Etc/GMT-5
Etc/GMT+6
Etc/GMT-6
Etc/GMT+7
Etc/GMT-7
Etc/GMT+8
Etc/GMT-8
Etc/GMT+9
Etc/GMT-9
GMT+0
GMT-0
```

filimonov replied:
Also - not sure if TZ with plus / minus char in name will work properly from binary

filimonov replied:
See also 
https://github.com/ClickHouse/ClickHouse/issues/4443
https://github.com/ClickHouse/ClickHouse/issues/4421

alexey-milovidov replied:
Autogeneration is good, I wanted to do it initially.

About GMT+- - I expect they won't work - then we should remove them from the list to avoid keeping them as trash.

alexey-milovidov replied:
It's better to get the list of timezones from CCTZ, otherwise it's too easy to build ClickHouse with outdated tzdata from system. People tend to not update the OS packages when not necessarily.

Or: you can somehow decide what is newer.

alexey-milovidov replied:
Yes, you have to remove some
```
2020-08-12 15:52:52 ld.lld-10: error: duplicate symbol: _binary_Etc_GMT_0_start
2020-08-12 15:52:52 >>> defined at Etc_GMT+0.o:(.rodata+0x0) in archive /ClickHouse/build/contrib/cctz-cmake/libtzdata.a
2020-08-12 15:52:52 >>> defined at Etc_GMT-0.o:(.rodata+0x0) in archive /ClickHouse/build/contrib/cctz-cmake/libtzdata.a
```

filimonov replied:
> About GMT+- - I expect they won't work -

They work if tzdata installed in system:
https://github.com/ClickHouse/ClickHouse/blob/0f79eb3cc518977a46b8020c6e317cae15ff27b5/base/common/DateLUTImpl.cpp#L218-L219

 WIll try to fix them (want to have them working properly from binary).

> It's better to get the list of timezones from CCTZ

Agree and it's already done this way (so updating tzdata package is not needed and i removed that).

So we only need to remember about updating cctz itself (i've added a simple bash script to check if update is needed - you can add it in CI).

filimonov commented:
Are we sure it works at all? Some symbol names looks cropped:
```
readelf -a /usr/bin/clickhouse | grep _binary | cut -c 60- | sort 
_binary_Africa_Abidjan_en
_binary_Africa_Abidjan_si
_binary_Africa_Abidjan_st
_binary_Africa_Accra_end
_binary_Africa_Accra_size
_binary_Africa_Accra_star
_binary_Africa_Addis_Abab
_binary_Africa_Addis_Abab
_binary_Africa_Addis_Abab
_binary_Africa_Algiers_en
_binary_Africa_Algiers_si
_binary_Africa_Algiers_st
_binary_Africa_Asmara_end
_binary_Africa_Asmara_siz
_binary_Africa_Asmara_sta
_binary_Africa_Asmera_end
_binary_Africa_Asmera_siz
_binary_Africa_Asmera_sta
_binary_Africa_Bamako_end
_binary_Africa_Bamako_siz
_binary_Africa_Bamako_sta
_binary_Africa_Bangui_end
_binary_Africa_Bangui_siz
_binary_Africa_Bangui_sta
_binary_Africa_Banjul_end
_binary_Africa_Banjul_siz
_binary_Africa_Banjul_sta
_binary_Africa_Bissau_end
_binary_Africa_Bissau_siz
_binary_Africa_Bissau_sta
_binary_Africa_Blantyre_e
_binary_Africa_Blantyre_s
_binary_Africa_Blantyre_s
_binary_Africa_Brazzavill
_binary_Africa_Brazzavill
_binary_Africa_Brazzavill
_binary_Africa_Bujumbura_
_binary_Africa_Bujumbura_
_binary_Africa_Bujumbura_
_binary_Africa_Cairo_end
_binary_Africa_Cairo_size
_binary_Africa_Cairo_star
_binary_Africa_Casablanca
_binary_Africa_Casablanca
_binary_Africa_Casablanca
_binary_Africa_Ceuta_end
_binary_Africa_Ceuta_size
_binary_Africa_Ceuta_star
_binary_Africa_Conakry_en
_binary_Africa_Conakry_si
_binary_Africa_Conakry_st
_binary_Africa_Dakar_end
_binary_Africa_Dakar_size
_binary_Africa_Dakar_star
_binary_Africa_Dar_es_Sal
_binary_Africa_Dar_es_Sal
_binary_Africa_Dar_es_Sal
_binary_Africa_Djibouti_e
_binary_Africa_Djibouti_s
_binary_Africa_Djibouti_s
_binary_Africa_Douala_end
_binary_Africa_Douala_siz
_binary_Africa_Douala_sta
_binary_Africa_El_Aaiun_e
_binary_Africa_El_Aaiun_s
_binary_Africa_El_Aaiun_s
_binary_Africa_Freetown_e
_binary_Africa_Freetown_s
_binary_Africa_Freetown_s
_binary_Africa_Gaborone_e
_binary_Africa_Gaborone_s
_binary_Africa_Gaborone_s
_binary_Africa_Harare_end
_binary_Africa_Harare_siz
_binary_Africa_Harare_sta
_binary_Africa_Johannesbu
_binary_Africa_Johannesbu
_binary_Africa_Johannesbu
_binary_Africa_Juba_end
_binary_Africa_Juba_size
_binary_Africa_Juba_start
_binary_Africa_Kampala_en
_binary_Africa_Kampala_si
_binary_Africa_Kampala_st
_binary_Africa_Khartoum_e
_binary_Africa_Khartoum_s
_binary_Africa_Khartoum_s
_binary_Africa_Kigali_end
_binary_Africa_Kigali_siz
_binary_Africa_Kigali_sta
_binary_Africa_Kinshasa_e
_binary_Africa_Kinshasa_s
_binary_Africa_Kinshasa_s
_binary_Africa_Lagos_end
_binary_Africa_Lagos_size
_binary_Africa_Lagos_star
_binary_Africa_Libreville
_binary_Africa_Libreville
_binary_Africa_Libreville
_binary_Africa_Lome_end
_binary_Africa_Lome_size
_binary_Africa_Lome_start
_binary_Africa_Luanda_end
_binary_Africa_Luanda_siz
_binary_Africa_Luanda_sta
_binary_Africa_Lubumbashi
_binary_Africa_Lubumbashi
_binary_Africa_Lubumbashi
_binary_Africa_Lusaka_end
_binary_Africa_Lusaka_siz
_binary_Africa_Lusaka_sta
_binary_Africa_Malabo_end
_binary_Africa_Malabo_siz
_binary_Africa_Malabo_sta
_binary_Africa_Maputo_end
_binary_Africa_Maputo_siz
_binary_Africa_Maputo_sta
_binary_Africa_Maseru_end
_binary_Africa_Maseru_siz
_binary_Africa_Maseru_sta
_binary_Africa_Mbabane_en
_binary_Africa_Mbabane_si
_binary_Africa_Mbabane_st
_binary_Africa_Mogadishu_
_binary_Africa_Mogadishu_
_binary_Africa_Mogadishu_
_binary_Africa_Monrovia_e
_binary_Africa_Monrovia_s
_binary_Africa_Monrovia_s
_binary_Africa_Nairobi_en
_binary_Africa_Nairobi_si
_binary_Africa_Nairobi_st
_binary_Africa_Ndjamena_e
_binary_Africa_Ndjamena_s
_binary_Africa_Ndjamena_s
_binary_Africa_Niamey_end
_binary_Africa_Niamey_siz
_binary_Africa_Niamey_sta
_binary_Africa_Nouakchott
_binary_Africa_Nouakchott
_binary_Africa_Nouakchott
_binary_Africa_Ouagadougo
_binary_Africa_Ouagadougo
_binary_Africa_Ouagadougo
_binary_Africa_Porto_Novo
_binary_Africa_Porto_Novo
_binary_Africa_Porto_Novo
_binary_Africa_Sao_Tome_e
_binary_Africa_Sao_Tome_s
_binary_Africa_Sao_Tome_s
_binary_Africa_Timbuktu_e
_binary_Africa_Timbuktu_s
_binary_Africa_Timbuktu_s
_binary_Africa_Tripoli_en
_binary_Africa_Tripoli_si
_binary_Africa_Tripoli_st
_binary_Africa_Tunis_end
_binary_Africa_Tunis_size
_binary_Africa_Tunis_star
_binary_Africa_Windhoek_e
_binary_Africa_Windhoek_s
_binary_Africa_Windhoek_s
_binary_America_Adak_end
_binary_America_Adak_size
_binary_America_Adak_star
_binary_America_Anchorage
_binary_America_Anchorage
_binary_America_Anchorage
_binary_America_Anguilla_
_binary_America_Anguilla_
_binary_America_Anguilla_
_binary_America_Antigua_e
_binary_America_Antigua_s
_binary_America_Antigua_s
_binary_America_Araguaina
_binary_America_Araguaina
_binary_America_Araguaina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Argentina
_binary_America_Aruba_end
_binary_America_Aruba_siz
_binary_America_Aruba_sta
_binary_America_Asuncion_
_binary_America_Asuncion_
_binary_America_Asuncion_
_binary_America_Atikokan_
_binary_America_Atikokan_
_binary_America_Atikokan_
_binary_America_Atka_end
_binary_America_Atka_size
_binary_America_Atka_star
_binary_America_Bahia_Ban
_binary_America_Bahia_Ban
_binary_America_Bahia_Ban
_binary_America_Bahia_end
_binary_America_Bahia_siz
_binary_America_Bahia_sta
_binary_America_Barbados_
_binary_America_Barbados_
_binary_America_Barbados_
_binary_America_Belem_end
_binary_America_Belem_siz
_binary_America_Belem_sta
_binary_America_Belize_en
_binary_America_Belize_si
_binary_America_Belize_st
_binary_America_Blanc_Sab
_binary_America_Blanc_Sab
_binary_America_Blanc_Sab
_binary_America_Boa_Vista
_binary_America_Boa_Vista
_binary_America_Boa_Vista
_binary_America_Bogota_en
_binary_America_Bogota_si
_binary_America_Bogota_st
_binary_America_Boise_end
_binary_America_Boise_siz
_binary_America_Boise_sta
_binary_America_Buenos_Ai
_binary_America_Buenos_Ai
_binary_America_Buenos_Ai
_binary_America_Cambridge
_binary_America_Cambridge
_binary_America_Cambridge
_binary_America_Campo_Gra
_binary_America_Campo_Gra
_binary_America_Campo_Gra
_binary_America_Cancun_en
_binary_America_Cancun_si
_binary_America_Cancun_st
_binary_America_Caracas_e
_binary_America_Caracas_s
_binary_America_Caracas_s
_binary_America_Catamarca
_binary_America_Catamarca
_binary_America_Catamarca
_binary_America_Cayenne_e
_binary_America_Cayenne_s
_binary_America_Cayenne_s
_binary_America_Cayman_en
_binary_America_Cayman_si
_binary_America_Cayman_st
_binary_America_Chicago_e
_binary_America_Chicago_s
_binary_America_Chicago_s
_binary_America_Chihuahua
_binary_America_Chihuahua
_binary_America_Chihuahua
_binary_America_Coral_Har
_binary_America_Coral_Har
_binary_America_Coral_Har
_binary_America_Cordoba_e
_binary_America_Cordoba_s
_binary_America_Cordoba_s
_binary_America_Costa_Ric
_binary_America_Costa_Ric
_binary_America_Costa_Ric
_binary_America_Creston_e
_binary_America_Creston_s
_binary_America_Creston_s
_binary_America_Cuiaba_en
_binary_America_Cuiaba_si
_binary_America_Cuiaba_st
_binary_America_Curacao_e
_binary_America_Curacao_s
_binary_America_Curacao_s
_binary_America_Danmarksh
_binary_America_Danmarksh
_binary_America_Danmarksh
_binary_America_Dawson_Cr
_binary_America_Dawson_Cr
_binary_America_Dawson_Cr
_binary_America_Dawson_en
_binary_America_Dawson_si
_binary_America_Dawson_st
_binary_America_Denver_en
_binary_America_Denver_si
_binary_America_Denver_st
_binary_America_Detroit_e
_binary_America_Detroit_s
_binary_America_Detroit_s
_binary_America_Dominica_
_binary_America_Dominica_
_binary_America_Dominica_
_binary_America_Edmonton_
_binary_America_Edmonton_
_binary_America_Edmonton_
_binary_America_Eirunepe_
_binary_America_Eirunepe_
_binary_America_Eirunepe_
_binary_America_El_Salvad
_binary_America_El_Salvad
_binary_America_El_Salvad
_binary_America_Ensenada_
_binary_America_Ensenada_
_binary_America_Ensenada_
_binary_America_Fortaleza
_binary_America_Fortaleza
_binary_America_Fortaleza
_binary_America_Fort_Nels
_binary_America_Fort_Nels
_binary_America_Fort_Nels
_binary_America_Fort_Wayn
_binary_America_Fort_Wayn
_binary_America_Fort_Wayn
_binary_America_Glace_Bay
_binary_America_Glace_Bay
_binary_America_Glace_Bay
_binary_America_Godthab_e
_binary_America_Godthab_s
_binary_America_Godthab_s
_binary_America_Goose_Bay
_binary_America_Goose_Bay
_binary_America_Goose_Bay
_binary_America_Grand_Tur
_binary_America_Grand_Tur
_binary_America_Grand_Tur
_binary_America_Grenada_e
_binary_America_Grenada_s
_binary_America_Grenada_s
_binary_America_Guadeloup
_binary_America_Guadeloup
_binary_America_Guadeloup
_binary_America_Guatemala
_binary_America_Guatemala
_binary_America_Guatemala
_binary_America_Guayaquil
_binary_America_Guayaquil
_binary_America_Guayaquil
_binary_America_Guyana_en
_binary_America_Guyana_si
_binary_America_Guyana_st
_binary_America_Halifax_e
_binary_America_Halifax_s
_binary_America_Halifax_s
_binary_America_Havana_en
_binary_America_Havana_si
_binary_America_Havana_st
_binary_America_Hermosill
_binary_America_Hermosill
_binary_America_Hermosill
_binary_America_Indiana_I
_binary_America_Indiana_I
_binary_America_Indiana_I
_binary_America_Indiana_K
_binary_America_Indiana_K
_binary_America_Indiana_K
_binary_America_Indiana_M
_binary_America_Indiana_M
_binary_America_Indiana_M
_binary_America_Indiana_P
_binary_America_Indiana_P
_binary_America_Indiana_P
_binary_America_Indianapo
_binary_America_Indianapo
_binary_America_Indianapo
_binary_America_Indiana_T
_binary_America_Indiana_T
_binary_America_Indiana_T
_binary_America_Indiana_V
_binary_America_Indiana_V
_binary_America_Indiana_V
_binary_America_Indiana_V
_binary_America_Indiana_V
_binary_America_Indiana_V
_binary_America_Indiana_W
_binary_America_Indiana_W
_binary_America_Indiana_W
_binary_America_Inuvik_en
_binary_America_Inuvik_si
_binary_America_Inuvik_st
_binary_America_Iqaluit_e
_binary_America_Iqaluit_s
_binary_America_Iqaluit_s
_binary_America_Jamaica_e
_binary_America_Jamaica_s
_binary_America_Jamaica_s
_binary_America_Jujuy_end
_binary_America_Jujuy_siz
_binary_America_Jujuy_sta
_binary_America_Juneau_en
_binary_America_Juneau_si
_binary_America_Juneau_st
_binary_America_Kentucky_
_binary_America_Kentucky_
_binary_America_Kentucky_
_binary_America_Kentucky_
_binary_America_Kentucky_
_binary_America_Kentucky_
_binary_America_Knox_IN_e
_binary_America_Knox_IN_s
_binary_America_Knox_IN_s
_binary_America_Kralendij
_binary_America_Kralendij
_binary_America_Kralendij
_binary_America_La_Paz_en
_binary_America_La_Paz_si
_binary_America_La_Paz_st
_binary_America_Lima_end
_binary_America_Lima_size
_binary_America_Lima_star
_binary_America_Los_Angel
_binary_America_Los_Angel
_binary_America_Los_Angel
_binary_America_Louisvill
_binary_America_Louisvill
_binary_America_Louisvill
_binary_America_Lower_Pri
_binary_America_Lower_Pri
_binary_America_Lower_Pri
_binary_America_Maceio_en
_binary_America_Maceio_si
_binary_America_Maceio_st
_binary_America_Managua_e
_binary_America_Managua_s
_binary_America_Managua_s
_binary_America_Manaus_en
_binary_America_Manaus_si
_binary_America_Manaus_st
_binary_America_Marigot_e
_binary_America_Marigot_s
_binary_America_Marigot_s
_binary_America_Martiniqu
_binary_America_Martiniqu
_binary_America_Martiniqu
_binary_America_Matamoros
_binary_America_Matamoros
_binary_America_Matamoros
_binary_America_Mazatlan_
_binary_America_Mazatlan_
_binary_America_Mazatlan_
_binary_America_Mendoza_e
_binary_America_Mendoza_s
_binary_America_Mendoza_s
_binary_America_Menominee
_binary_America_Menominee
_binary_America_Menominee
_binary_America_Merida_en
_binary_America_Merida_si
_binary_America_Merida_st
_binary_America_Metlakatl
_binary_America_Metlakatl
_binary_America_Metlakatl
_binary_America_Mexico_Ci
_binary_America_Mexico_Ci
_binary_America_Mexico_Ci
_binary_America_Miquelon_
_binary_America_Miquelon_
_binary_America_Miquelon_
_binary_America_Moncton_e
_binary_America_Moncton_s
_binary_America_Moncton_s
_binary_America_Monterrey
_binary_America_Monterrey
_binary_America_Monterrey
_binary_America_Montevide
_binary_America_Montevide
_binary_America_Montevide
_binary_America_Montreal_
_binary_America_Montreal_
_binary_America_Montreal_
_binary_America_Montserra
_binary_America_Montserra
_binary_America_Montserra
_binary_America_Nassau_en
_binary_America_Nassau_si
_binary_America_Nassau_st
_binary_America_New_York_
_binary_America_New_York_
_binary_America_New_York_
_binary_America_Nipigon_e
_binary_America_Nipigon_s
_binary_America_Nipigon_s
_binary_America_Nome_end
_binary_America_Nome_size
_binary_America_Nome_star
_binary_America_Noronha_e
_binary_America_Noronha_s
_binary_America_Noronha_s
_binary_America_North_Dak
_binary_America_North_Dak
_binary_America_North_Dak
_binary_America_North_Dak
_binary_America_North_Dak
_binary_America_North_Dak
_binary_America_North_Dak
_binary_America_North_Dak
_binary_America_North_Dak
_binary_America_Ojinaga_e
_binary_America_Ojinaga_s
_binary_America_Ojinaga_s
_binary_America_Panama_en
_binary_America_Panama_si
_binary_America_Panama_st
_binary_America_Pangnirtu
_binary_America_Pangnirtu
_binary_America_Pangnirtu
_binary_America_Paramarib
_binary_America_Paramarib
_binary_America_Paramarib
_binary_America_Phoenix_e
_binary_America_Phoenix_s
_binary_America_Phoenix_s
_binary_America_Port_au_P
_binary_America_Port_au_P
_binary_America_Port_au_P
_binary_America_Porto_Acr
_binary_America_Porto_Acr
_binary_America_Porto_Acr
_binary_America_Port_of_S
_binary_America_Port_of_S
_binary_America_Port_of_S
_binary_America_Porto_Vel
_binary_America_Porto_Vel
_binary_America_Porto_Vel
_binary_America_Puerto_Ri
_binary_America_Puerto_Ri
_binary_America_Puerto_Ri
_binary_America_Punta_Are
_binary_America_Punta_Are
_binary_America_Punta_Are
_binary_America_Rainy_Riv
_binary_America_Rainy_Riv
_binary_America_Rainy_Riv
_binary_America_Rankin_In
_binary_America_Rankin_In
_binary_America_Rankin_In
_binary_America_Recife_en
_binary_America_Recife_si
_binary_America_Recife_st
_binary_America_Regina_en
_binary_America_Regina_si
_binary_America_Regina_st
_binary_America_Resolute_
_binary_America_Resolute_
_binary_America_Resolute_
_binary_America_Rio_Branc
_binary_America_Rio_Branc
_binary_America_Rio_Branc
_binary_America_Rosario_e
_binary_America_Rosario_s
_binary_America_Rosario_s
_binary_America_Santa_Isa
_binary_America_Santa_Isa
_binary_America_Santa_Isa
_binary_America_Santarem_
_binary_America_Santarem_
_binary_America_Santarem_
_binary_America_Santiago_
_binary_America_Santiago_
_binary_America_Santiago_
_binary_America_Santo_Dom
_binary_America_Santo_Dom
_binary_America_Santo_Dom
_binary_America_Sao_Paulo
_binary_America_Sao_Paulo
_binary_America_Sao_Paulo
_binary_America_Scoresbys
_binary_America_Scoresbys
_binary_America_Scoresbys
_binary_America_Shiprock_
_binary_America_Shiprock_
_binary_America_Shiprock_
_binary_America_Sitka_end
_binary_America_Sitka_siz
_binary_America_Sitka_sta
_binary_America_St_Barthe
_binary_America_St_Barthe
_binary_America_St_Barthe
_binary_America_St_Johns_
_binary_America_St_Johns_
_binary_America_St_Johns_
_binary_America_St_Kitts_
_binary_America_St_Kitts_
_binary_America_St_Kitts_
_binary_America_St_Lucia_
_binary_America_St_Lucia_
_binary_America_St_Lucia_
_binary_America_St_Thomas
_binary_America_St_Thomas
_binary_America_St_Thomas
_binary_America_St_Vincen
_binary_America_St_Vincen
_binary_America_St_Vincen
_binary_America_Swift_Cur
_binary_America_Swift_Cur
_binary_America_Swift_Cur
_binary_America_Tegucigal
_binary_America_Tegucigal
_binary_America_Tegucigal
_binary_America_Thule_end
_binary_America_Thule_siz
_binary_America_Thule_sta
_binary_America_Thunder_B
_binary_America_Thunder_B
_binary_America_Thunder_B
_binary_America_Tijuana_e
_binary_America_Tijuana_s
_binary_America_Tijuana_s
_binary_America_Toronto_e
_binary_America_Toronto_s
_binary_America_Toronto_s
_binary_America_Tortola_e
_binary_America_Tortola_s
_binary_America_Tortola_s
_binary_America_Vancouver
_binary_America_Vancouver
_binary_America_Vancouver
_binary_America_Virgin_en
_binary_America_Virgin_si
_binary_America_Virgin_st
_binary_America_Whitehors
_binary_America_Whitehors
_binary_America_Whitehors
_binary_America_Winnipeg_
_binary_America_Winnipeg_
_binary_America_Winnipeg_
_binary_America_Yakutat_e
_binary_America_Yakutat_s
_binary_America_Yakutat_s
_binary_America_Yellowkni
_binary_America_Yellowkni
_binary_America_Yellowkni
_binary_Antarctica_Casey_
_binary_Antarctica_Casey_
_binary_Antarctica_Casey_
_binary_Antarctica_Davis_
_binary_Antarctica_Davis_
_binary_Antarctica_Davis_
_binary_Antarctica_Dumont
_binary_Antarctica_Dumont
_binary_Antarctica_Dumont
_binary_Antarctica_Macqua
_binary_Antarctica_Macqua
_binary_Antarctica_Macqua
_binary_Antarctica_Mawson
_binary_Antarctica_Mawson
_binary_Antarctica_Mawson
_binary_Antarctica_McMurd
_binary_Antarctica_McMurd
_binary_Antarctica_McMurd
_binary_Antarctica_Palmer
_binary_Antarctica_Palmer
_binary_Antarctica_Palmer
_binary_Antarctica_Rother
_binary_Antarctica_Rother
_binary_Antarctica_Rother
_binary_Antarctica_South_
_binary_Antarctica_South_
_binary_Antarctica_South_
_binary_Antarctica_Syowa_
_binary_Antarctica_Syowa_
_binary_Antarctica_Syowa_
_binary_Antarctica_Troll_
_binary_Antarctica_Troll_
_binary_Antarctica_Troll_
_binary_Antarctica_Vostok
_binary_Antarctica_Vostok
_binary_Antarctica_Vostok
_binary_Arctic_Longyearby
_binary_Arctic_Longyearby
_binary_Arctic_Longyearby
_binary_Asia_Aden_end
_binary_Asia_Aden_size
_binary_Asia_Aden_start
_binary_Asia_Almaty_end
_binary_Asia_Almaty_size
_binary_Asia_Almaty_start
_binary_Asia_Amman_end
_binary_Asia_Amman_size
_binary_Asia_Amman_start
_binary_Asia_Anadyr_end
_binary_Asia_Anadyr_size
_binary_Asia_Anadyr_start
_binary_Asia_Aqtau_end
_binary_Asia_Aqtau_size
_binary_Asia_Aqtau_start
_binary_Asia_Aqtobe_end
_binary_Asia_Aqtobe_size
_binary_Asia_Aqtobe_start
_binary_Asia_Ashgabat_end
_binary_Asia_Ashgabat_siz
_binary_Asia_Ashgabat_sta
_binary_Asia_Ashkhabad_en
_binary_Asia_Ashkhabad_si
_binary_Asia_Ashkhabad_st
_binary_Asia_Atyrau_end
_binary_Asia_Atyrau_size
_binary_Asia_Atyrau_start
_binary_Asia_Baghdad_end
_binary_Asia_Baghdad_size
_binary_Asia_Baghdad_star
_binary_Asia_Bahrain_end
_binary_Asia_Bahrain_size
_binary_Asia_Bahrain_star
_binary_Asia_Baku_end
_binary_Asia_Baku_size
_binary_Asia_Baku_start
_binary_Asia_Bangkok_end
_binary_Asia_Bangkok_size
_binary_Asia_Bangkok_star
_binary_Asia_Barnaul_end
_binary_Asia_Barnaul_size
_binary_Asia_Barnaul_star
_binary_Asia_Beirut_end
_binary_Asia_Beirut_size
_binary_Asia_Beirut_start
_binary_Asia_Bishkek_end
_binary_Asia_Bishkek_size
_binary_Asia_Bishkek_star
_binary_Asia_Brunei_end
_binary_Asia_Brunei_size
_binary_Asia_Brunei_start
_binary_Asia_Calcutta_end
_binary_Asia_Calcutta_siz
_binary_Asia_Calcutta_sta
_binary_Asia_Chita_end
_binary_Asia_Chita_size
_binary_Asia_Chita_start
_binary_Asia_Choibalsan_e
_binary_Asia_Choibalsan_s
_binary_Asia_Choibalsan_s
_binary_Asia_Chongqing_en
_binary_Asia_Chongqing_si
_binary_Asia_Chongqing_st
_binary_Asia_Chungking_en
_binary_Asia_Chungking_si
_binary_Asia_Chungking_st
_binary_Asia_Colombo_end
_binary_Asia_Colombo_size
_binary_Asia_Colombo_star
_binary_Asia_Dacca_end
_binary_Asia_Dacca_size
_binary_Asia_Dacca_start
_binary_Asia_Damascus_end
_binary_Asia_Damascus_siz
_binary_Asia_Damascus_sta
_binary_Asia_Dhaka_end
_binary_Asia_Dhaka_size
_binary_Asia_Dhaka_start
_binary_Asia_Dili_end
_binary_Asia_Dili_size
_binary_Asia_Dili_start
_binary_Asia_Dubai_end
_binary_Asia_Dubai_size
_binary_Asia_Dubai_start
_binary_Asia_Dushanbe_end
_binary_Asia_Dushanbe_siz
_binary_Asia_Dushanbe_sta
_binary_Asia_Famagusta_en
_binary_Asia_Famagusta_si
_binary_Asia_Famagusta_st
_binary_Asia_Gaza_end
_binary_Asia_Gaza_size
_binary_Asia_Gaza_start
_binary_Asia_Harbin_end
_binary_Asia_Harbin_size
_binary_Asia_Harbin_start
_binary_Asia_Hebron_end
_binary_Asia_Hebron_size
_binary_Asia_Hebron_start
_binary_Asia_Ho_Chi_Minh_
_binary_Asia_Ho_Chi_Minh_
_binary_Asia_Ho_Chi_Minh_
_binary_Asia_Hong_Kong_en
_binary_Asia_Hong_Kong_si
_binary_Asia_Hong_Kong_st
_binary_Asia_Hovd_end
_binary_Asia_Hovd_size
_binary_Asia_Hovd_start
_binary_Asia_Irkutsk_end
_binary_Asia_Irkutsk_size
_binary_Asia_Irkutsk_star
_binary_Asia_Istanbul_end
_binary_Asia_Istanbul_siz
_binary_Asia_Istanbul_sta
_binary_Asia_Jakarta_end
_binary_Asia_Jakarta_size
_binary_Asia_Jakarta_star
_binary_Asia_Jayapura_end
_binary_Asia_Jayapura_siz
_binary_Asia_Jayapura_sta
_binary_Asia_Jerusalem_en
_binary_Asia_Jerusalem_si
_binary_Asia_Jerusalem_st
_binary_Asia_Kabul_end
_binary_Asia_Kabul_size
_binary_Asia_Kabul_start
_binary_Asia_Kamchatka_en
_binary_Asia_Kamchatka_si
_binary_Asia_Kamchatka_st
_binary_Asia_Karachi_end
_binary_Asia_Karachi_size
_binary_Asia_Karachi_star
_binary_Asia_Kashgar_end
_binary_Asia_Kashgar_size
_binary_Asia_Kashgar_star
_binary_Asia_Kathmandu_en
_binary_Asia_Kathmandu_si
_binary_Asia_Kathmandu_st
_binary_Asia_Katmandu_end
_binary_Asia_Katmandu_siz
_binary_Asia_Katmandu_sta
_binary_Asia_Khandyga_end
_binary_Asia_Khandyga_siz
_binary_Asia_Khandyga_sta
_binary_Asia_Kolkata_end
_binary_Asia_Kolkata_size
_binary_Asia_Kolkata_star
_binary_Asia_Krasnoyarsk_
_binary_Asia_Krasnoyarsk_
_binary_Asia_Krasnoyarsk_
_binary_Asia_Kuala_Lumpur
_binary_Asia_Kuala_Lumpur
_binary_Asia_Kuala_Lumpur
_binary_Asia_Kuching_end
_binary_Asia_Kuching_size
_binary_Asia_Kuching_star
_binary_Asia_Kuwait_end
_binary_Asia_Kuwait_size
_binary_Asia_Kuwait_start
_binary_Asia_Macao_end
_binary_Asia_Macao_size
_binary_Asia_Macao_start
_binary_Asia_Macau_end
_binary_Asia_Macau_size
_binary_Asia_Macau_start
_binary_Asia_Magadan_end
_binary_Asia_Magadan_size
_binary_Asia_Magadan_star
_binary_Asia_Makassar_end
_binary_Asia_Makassar_siz
_binary_Asia_Makassar_sta
_binary_Asia_Manila_end
_binary_Asia_Manila_size
_binary_Asia_Manila_start
_binary_Asia_Muscat_end
_binary_Asia_Muscat_size
_binary_Asia_Muscat_start
_binary_Asia_Nicosia_end
_binary_Asia_Nicosia_size
_binary_Asia_Nicosia_star
_binary_Asia_Novokuznetsk
_binary_Asia_Novokuznetsk
_binary_Asia_Novokuznetsk
_binary_Asia_Novosibirsk_
_binary_Asia_Novosibirsk_
_binary_Asia_Novosibirsk_
_binary_Asia_Omsk_end
_binary_Asia_Omsk_size
_binary_Asia_Omsk_start
_binary_Asia_Oral_end
_binary_Asia_Oral_size
_binary_Asia_Oral_start
_binary_Asia_Phnom_Penh_e
_binary_Asia_Phnom_Penh_s
_binary_Asia_Phnom_Penh_s
_binary_Asia_Pontianak_en
_binary_Asia_Pontianak_si
_binary_Asia_Pontianak_st
_binary_Asia_Pyongyang_en
_binary_Asia_Pyongyang_si
_binary_Asia_Pyongyang_st
_binary_Asia_Qatar_end
_binary_Asia_Qatar_size
_binary_Asia_Qatar_start
_binary_Asia_Qostanay_end
_binary_Asia_Qostanay_siz
_binary_Asia_Qostanay_sta
_binary_Asia_Qyzylorda_en
_binary_Asia_Qyzylorda_si
_binary_Asia_Qyzylorda_st
_binary_Asia_Rangoon_end
_binary_Asia_Rangoon_size
_binary_Asia_Rangoon_star
_binary_Asia_Riyadh_end
_binary_Asia_Riyadh_size
_binary_Asia_Riyadh_start
_binary_Asia_Saigon_end
_binary_Asia_Saigon_size
_binary_Asia_Saigon_start
_binary_Asia_Sakhalin_end
_binary_Asia_Sakhalin_siz
_binary_Asia_Sakhalin_sta
_binary_Asia_Samarkand_en
_binary_Asia_Samarkand_si
_binary_Asia_Samarkand_st
_binary_Asia_Seoul_end
_binary_Asia_Seoul_size
_binary_Asia_Seoul_start
_binary_Asia_Shanghai_end
_binary_Asia_Shanghai_siz
_binary_Asia_Shanghai_sta
_binary_Asia_Singapore_en
_binary_Asia_Singapore_si
_binary_Asia_Singapore_st
_binary_Asia_Srednekolyms
_binary_Asia_Srednekolyms
_binary_Asia_Srednekolyms
_binary_Asia_Taipei_end
_binary_Asia_Taipei_size
_binary_Asia_Taipei_start
_binary_Asia_Tashkent_end
_binary_Asia_Tashkent_siz
_binary_Asia_Tashkent_sta
_binary_Asia_Tbilisi_end
_binary_Asia_Tbilisi_size
_binary_Asia_Tbilisi_star
_binary_Asia_Tehran_end
_binary_Asia_Tehran_size
_binary_Asia_Tehran_start
_binary_Asia_Tel_Aviv_end
_binary_Asia_Tel_Aviv_siz
_binary_Asia_Tel_Aviv_sta
_binary_Asia_Thimbu_end
_binary_Asia_Thimbu_size
_binary_Asia_Thimbu_start
_binary_Asia_Thimphu_end
_binary_Asia_Thimphu_size
_binary_Asia_Thimphu_star
_binary_Asia_Tokyo_end
_binary_Asia_Tokyo_size
_binary_Asia_Tokyo_start
_binary_Asia_Tomsk_end
_binary_Asia_Tomsk_size
_binary_Asia_Tomsk_start
_binary_Asia_Ujung_Pandan
_binary_Asia_Ujung_Pandan
_binary_Asia_Ujung_Pandan
_binary_Asia_Ulaanbaatar_
_binary_Asia_Ulaanbaatar_
_binary_Asia_Ulaanbaatar_
_binary_Asia_Ulan_Bator_e
_binary_Asia_Ulan_Bator_s
_binary_Asia_Ulan_Bator_s
_binary_Asia_Urumqi_end
_binary_Asia_Urumqi_size
_binary_Asia_Urumqi_start
_binary_Asia_Ust_Nera_end
_binary_Asia_Ust_Nera_siz
_binary_Asia_Ust_Nera_sta
_binary_Asia_Vientiane_en
_binary_Asia_Vientiane_si
_binary_Asia_Vientiane_st
_binary_Asia_Vladivostok_
_binary_Asia_Vladivostok_
_binary_Asia_Vladivostok_
_binary_Asia_Yakutsk_end
_binary_Asia_Yakutsk_size
_binary_Asia_Yakutsk_star
_binary_Asia_Yangon_end
_binary_Asia_Yangon_size
_binary_Asia_Yangon_start
_binary_Asia_Yekaterinbur
_binary_Asia_Yekaterinbur
_binary_Asia_Yekaterinbur
_binary_Asia_Yerevan_end
_binary_Asia_Yerevan_size
_binary_Asia_Yerevan_star
_binary_Atlantic_Azores_e
_binary_Atlantic_Azores_s
_binary_Atlantic_Azores_s
_binary_Atlantic_Bermuda_
_binary_Atlantic_Bermuda_
_binary_Atlantic_Bermuda_
_binary_Atlantic_Canary_e
_binary_Atlantic_Canary_s
_binary_Atlantic_Canary_s
_binary_Atlantic_Cape_Ver
_binary_Atlantic_Cape_Ver
_binary_Atlantic_Cape_Ver
_binary_Atlantic_Faeroe_e
_binary_Atlantic_Faeroe_s
_binary_Atlantic_Faeroe_s
_binary_Atlantic_Faroe_en
_binary_Atlantic_Faroe_si
_binary_Atlantic_Faroe_st
_binary_Atlantic_Jan_Maye
_binary_Atlantic_Jan_Maye
_binary_Atlantic_Jan_Maye
_binary_Atlantic_Madeira_
_binary_Atlantic_Madeira_
_binary_Atlantic_Madeira_
_binary_Atlantic_Reykjavi
_binary_Atlantic_Reykjavi
_binary_Atlantic_Reykjavi
_binary_Atlantic_South_Ge
_binary_Atlantic_South_Ge
_binary_Atlantic_South_Ge
_binary_Atlantic_Stanley_
_binary_Atlantic_Stanley_
_binary_Atlantic_Stanley_
_binary_Atlantic_St_Helen
_binary_Atlantic_St_Helen
_binary_Atlantic_St_Helen
_binary_Australia_ACT_end
_binary_Australia_ACT_siz
_binary_Australia_ACT_sta
_binary_Australia_Adelaid
_binary_Australia_Adelaid
_binary_Australia_Adelaid
_binary_Australia_Brisban
_binary_Australia_Brisban
_binary_Australia_Brisban
_binary_Australia_Broken_
_binary_Australia_Broken_
_binary_Australia_Broken_
_binary_Australia_Canberr
_binary_Australia_Canberr
_binary_Australia_Canberr
_binary_Australia_Currie_
_binary_Australia_Currie_
_binary_Australia_Currie_
_binary_Australia_Darwin_
_binary_Australia_Darwin_
_binary_Australia_Darwin_
_binary_Australia_Eucla_e
_binary_Australia_Eucla_s
_binary_Australia_Eucla_s
_binary_Australia_Hobart_
_binary_Australia_Hobart_
_binary_Australia_Hobart_
_binary_Australia_LHI_end
_binary_Australia_LHI_siz
_binary_Australia_LHI_sta
_binary_Australia_Lindema
_binary_Australia_Lindema
_binary_Australia_Lindema
_binary_Australia_Lord_Ho
_binary_Australia_Lord_Ho
_binary_Australia_Lord_Ho
_binary_Australia_Melbour
_binary_Australia_Melbour
_binary_Australia_Melbour
_binary_Australia_North_e
_binary_Australia_North_s
_binary_Australia_North_s
_binary_Australia_NSW_end
_binary_Australia_NSW_siz
_binary_Australia_NSW_sta
_binary_Australia_Perth_e
_binary_Australia_Perth_s
_binary_Australia_Perth_s
_binary_Australia_Queensl
_binary_Australia_Queensl
_binary_Australia_Queensl
_binary_Australia_South_e
_binary_Australia_South_s
_binary_Australia_South_s
_binary_Australia_Sydney_
_binary_Australia_Sydney_
_binary_Australia_Sydney_
_binary_Australia_Tasmani
_binary_Australia_Tasmani
_binary_Australia_Tasmani
_binary_Australia_Victori
_binary_Australia_Victori
_binary_Australia_Victori
_binary_Australia_West_en
_binary_Australia_West_si
_binary_Australia_West_st
_binary_Australia_Yancowi
_binary_Australia_Yancowi
_binary_Australia_Yancowi
_binary_Brazil_Acre_end
_binary_Brazil_Acre_size
_binary_Brazil_Acre_start
_binary_Brazil_DeNoronha_
_binary_Brazil_DeNoronha_
_binary_Brazil_DeNoronha_
_binary_Brazil_East_end
_binary_Brazil_East_size
_binary_Brazil_East_start
_binary_Brazil_West_end
_binary_Brazil_West_size
_binary_Brazil_West_start
_binary_Canada_Atlantic_e
_binary_Canada_Atlantic_s
_binary_Canada_Atlantic_s
_binary_Canada_Central_en
_binary_Canada_Central_si
_binary_Canada_Central_st
_binary_Canada_Eastern_en
_binary_Canada_Eastern_si
_binary_Canada_Eastern_st
_binary_Canada_Mountain_e
_binary_Canada_Mountain_s
_binary_Canada_Mountain_s
_binary_Canada_Newfoundla
_binary_Canada_Newfoundla
_binary_Canada_Newfoundla
_binary_Canada_Pacific_en
_binary_Canada_Pacific_si
_binary_Canada_Pacific_st
_binary_Canada_Saskatchew
_binary_Canada_Saskatchew
_binary_Canada_Saskatchew
_binary_Canada_Yukon_end
_binary_Canada_Yukon_size
_binary_Canada_Yukon_star
_binary_CET_end
_binary_CET_size
_binary_CET_start
_binary_Chile_Continental
_binary_Chile_Continental
_binary_Chile_Continental
_binary_Chile_EasterIslan
_binary_Chile_EasterIslan
_binary_Chile_EasterIslan
_binary_CST6CDT_end
_binary_CST6CDT_size
_binary_CST6CDT_start
_binary_Cuba_end
_binary_Cuba_size
_binary_Cuba_start
_binary_EET_end
_binary_EET_size
_binary_EET_start
_binary_Egypt_end
_binary_Egypt_size
_binary_Egypt_start
_binary_Eire_end
_binary_Eire_size
_binary_Eire_start
_binary_EST5EDT_end
_binary_EST5EDT_size
_binary_EST5EDT_start
_binary_EST_end
_binary_EST_size
_binary_EST_start
_binary_Etc_GMT_end
_binary_Etc_GMT_size
_binary_Etc_GMT_start
_binary_Etc_Greenwich_end
_binary_Etc_Greenwich_siz
_binary_Etc_Greenwich_sta
_binary_Etc_UCT_end
_binary_Etc_UCT_size
_binary_Etc_UCT_start
_binary_Etc_Universal_end
_binary_Etc_Universal_siz
_binary_Etc_Universal_sta
_binary_Etc_UTC_end
_binary_Etc_UTC_size
_binary_Etc_UTC_start
_binary_Etc_Zulu_end
_binary_Etc_Zulu_size
_binary_Etc_Zulu_start
_binary_Europe_Amsterdam_
_binary_Europe_Amsterdam_
_binary_Europe_Amsterdam_
_binary_Europe_Andorra_en
_binary_Europe_Andorra_si
_binary_Europe_Andorra_st
_binary_Europe_Astrakhan_
_binary_Europe_Astrakhan_
_binary_Europe_Astrakhan_
_binary_Europe_Athens_end
_binary_Europe_Athens_siz
_binary_Europe_Athens_sta
_binary_Europe_Belfast_en
_binary_Europe_Belfast_si
_binary_Europe_Belfast_st
_binary_Europe_Belgrade_e
_binary_Europe_Belgrade_s
_binary_Europe_Belgrade_s
_binary_Europe_Berlin_end
_binary_Europe_Berlin_siz
_binary_Europe_Berlin_sta
_binary_Europe_Bratislava
_binary_Europe_Bratislava
_binary_Europe_Bratislava
_binary_Europe_Brussels_e
_binary_Europe_Brussels_s
_binary_Europe_Brussels_s
_binary_Europe_Bucharest_
_binary_Europe_Bucharest_
_binary_Europe_Bucharest_
_binary_Europe_Budapest_e
_binary_Europe_Budapest_s
_binary_Europe_Budapest_s
_binary_Europe_Busingen_e
_binary_Europe_Busingen_s
_binary_Europe_Busingen_s
_binary_Europe_Chisinau_e
_binary_Europe_Chisinau_s
_binary_Europe_Chisinau_s
_binary_Europe_Copenhagen
_binary_Europe_Copenhagen
_binary_Europe_Copenhagen
_binary_Europe_Dublin_end
_binary_Europe_Dublin_siz
_binary_Europe_Dublin_sta
_binary_Europe_Gibraltar_
_binary_Europe_Gibraltar_
_binary_Europe_Gibraltar_
_binary_Europe_Guernsey_e
_binary_Europe_Guernsey_s
_binary_Europe_Guernsey_s
_binary_Europe_Helsinki_e
_binary_Europe_Helsinki_s
_binary_Europe_Helsinki_s
_binary_Europe_Isle_of_Ma
_binary_Europe_Isle_of_Ma
_binary_Europe_Isle_of_Ma
_binary_Europe_Istanbul_e
_binary_Europe_Istanbul_s
_binary_Europe_Istanbul_s
_binary_Europe_Jersey_end
_binary_Europe_Jersey_siz
_binary_Europe_Jersey_sta
_binary_Europe_Kaliningra
_binary_Europe_Kaliningra
_binary_Europe_Kaliningra
_binary_Europe_Kiev_end
_binary_Europe_Kiev_size
_binary_Europe_Kiev_start
_binary_Europe_Kirov_end
_binary_Europe_Kirov_size
_binary_Europe_Kirov_star
_binary_Europe_Lisbon_end
_binary_Europe_Lisbon_siz
_binary_Europe_Lisbon_sta
_binary_Europe_Ljubljana_
_binary_Europe_Ljubljana_
_binary_Europe_Ljubljana_
_binary_Europe_London_end
_binary_Europe_London_siz
_binary_Europe_London_sta
_binary_Europe_Luxembourg
_binary_Europe_Luxembourg
_binary_Europe_Luxembourg
_binary_Europe_Madrid_end
_binary_Europe_Madrid_siz
_binary_Europe_Madrid_sta
_binary_Europe_Malta_end
_binary_Europe_Malta_size
_binary_Europe_Malta_star
_binary_Europe_Mariehamn_
_binary_Europe_Mariehamn_
_binary_Europe_Mariehamn_
_binary_Europe_Minsk_end
_binary_Europe_Minsk_size
_binary_Europe_Minsk_star
_binary_Europe_Monaco_end
_binary_Europe_Monaco_siz
_binary_Europe_Monaco_sta
_binary_Europe_Moscow_end
_binary_Europe_Moscow_siz
_binary_Europe_Moscow_sta
_binary_Europe_Nicosia_en
_binary_Europe_Nicosia_si
_binary_Europe_Nicosia_st
_binary_Europe_Oslo_end
_binary_Europe_Oslo_size
_binary_Europe_Oslo_start
_binary_Europe_Paris_end
_binary_Europe_Paris_size
_binary_Europe_Paris_star
_binary_Europe_Podgorica_
_binary_Europe_Podgorica_
_binary_Europe_Podgorica_
_binary_Europe_Prague_end
_binary_Europe_Prague_siz
_binary_Europe_Prague_sta
_binary_Europe_Riga_end
_binary_Europe_Riga_size
_binary_Europe_Riga_start
_binary_Europe_Rome_end
_binary_Europe_Rome_size
_binary_Europe_Rome_start
_binary_Europe_Samara_end
_binary_Europe_Samara_siz
_binary_Europe_Samara_sta
_binary_Europe_San_Marino
_binary_Europe_San_Marino
_binary_Europe_San_Marino
_binary_Europe_Sarajevo_e
_binary_Europe_Sarajevo_s
_binary_Europe_Sarajevo_s
_binary_Europe_Saratov_en
_binary_Europe_Saratov_si
_binary_Europe_Saratov_st
_binary_Europe_Simferopol
_binary_Europe_Simferopol
_binary_Europe_Simferopol
_binary_Europe_Skopje_end
_binary_Europe_Skopje_siz
_binary_Europe_Skopje_sta
_binary_Europe_Sofia_end
_binary_Europe_Sofia_size
_binary_Europe_Sofia_star
_binary_Europe_Stockholm_
_binary_Europe_Stockholm_
_binary_Europe_Stockholm_
_binary_Europe_Tallinn_en
_binary_Europe_Tallinn_si
_binary_Europe_Tallinn_st
_binary_Europe_Tirane_end
_binary_Europe_Tirane_siz
_binary_Europe_Tirane_sta
_binary_Europe_Tiraspol_e
_binary_Europe_Tiraspol_s
_binary_Europe_Tiraspol_s
_binary_Europe_Ulyanovsk_
_binary_Europe_Ulyanovsk_
_binary_Europe_Ulyanovsk_
_binary_Europe_Uzhgorod_e
_binary_Europe_Uzhgorod_s
_binary_Europe_Uzhgorod_s
_binary_Europe_Vaduz_end
_binary_Europe_Vaduz_size
_binary_Europe_Vaduz_star
_binary_Europe_Vatican_en
_binary_Europe_Vatican_si
_binary_Europe_Vatican_st
_binary_Europe_Vienna_end
_binary_Europe_Vienna_siz
_binary_Europe_Vienna_sta
_binary_Europe_Vilnius_en
_binary_Europe_Vilnius_si
_binary_Europe_Vilnius_st
_binary_Europe_Volgograd_
_binary_Europe_Volgograd_
_binary_Europe_Volgograd_
_binary_Europe_Warsaw_end
_binary_Europe_Warsaw_siz
_binary_Europe_Warsaw_sta
_binary_Europe_Zagreb_end
_binary_Europe_Zagreb_siz
_binary_Europe_Zagreb_sta
_binary_Europe_Zaporozhye
_binary_Europe_Zaporozhye
_binary_Europe_Zaporozhye
_binary_Europe_Zurich_end
_binary_Europe_Zurich_siz
_binary_Europe_Zurich_sta
_binary_Factory_end
_binary_Factory_size
_binary_Factory_start
_binary_GB_Eire_end
_binary_GB_Eire_size
_binary_GB_Eire_start
_binary_GB_end
_binary_GB_size
_binary_GB_start
_binary_GMT0_end
_binary_GMT0_size
_binary_GMT0_start
_binary_GMT_end
_binary_GMT_size
_binary_GMT_start
_binary_Greenwich_end
_binary_Greenwich_size
_binary_Greenwich_start
_binary_Hongkong_end
_binary_Hongkong_size
_binary_Hongkong_start
_binary_HST_end
_binary_HST_size
_binary_HST_start
_binary_Iceland_end
_binary_Iceland_size
_binary_Iceland_start
_binary_Indian_Antananari
_binary_Indian_Antananari
_binary_Indian_Antananari
_binary_Indian_Chagos_end
_binary_Indian_Chagos_siz
_binary_Indian_Chagos_sta
_binary_Indian_Christmas_
_binary_Indian_Christmas_
_binary_Indian_Christmas_
_binary_Indian_Cocos_end
_binary_Indian_Cocos_size
_binary_Indian_Cocos_star
_binary_Indian_Comoro_end
_binary_Indian_Comoro_siz
_binary_Indian_Comoro_sta
_binary_Indian_Kerguelen_
_binary_Indian_Kerguelen_
_binary_Indian_Kerguelen_
_binary_Indian_Mahe_end
_binary_Indian_Mahe_size
_binary_Indian_Mahe_start
_binary_Indian_Maldives_e
_binary_Indian_Maldives_s
_binary_Indian_Maldives_s
_binary_Indian_Mauritius_
_binary_Indian_Mauritius_
_binary_Indian_Mauritius_
_binary_Indian_Mayotte_en
_binary_Indian_Mayotte_si
_binary_Indian_Mayotte_st
_binary_Indian_Reunion_en
_binary_Indian_Reunion_si
_binary_Indian_Reunion_st
_binary_Iran_end
_binary_Iran_size
_binary_Iran_start
_binary_Israel_end
_binary_Israel_size
_binary_Israel_start
_binary_Jamaica_end
_binary_Jamaica_size
_binary_Jamaica_start
_binary_Japan_end
_binary_Japan_size
_binary_Japan_start
_binary_Kwajalein_end
_binary_Kwajalein_size
_binary_Kwajalein_start
_binary_Libya_end
_binary_Libya_size
_binary_Libya_start
_binary_MET_end
_binary_MET_size
_binary_MET_start
_binary_Mexico_BajaNorte_
_binary_Mexico_BajaNorte_
_binary_Mexico_BajaNorte_
_binary_Mexico_BajaSur_en
_binary_Mexico_BajaSur_si
_binary_Mexico_BajaSur_st
_binary_Mexico_General_en
_binary_Mexico_General_si
_binary_Mexico_General_st
_binary_MST7MDT_end
_binary_MST7MDT_size
_binary_MST7MDT_start
_binary_MST_end
_binary_MST_size
_binary_MST_start
_binary_Navajo_end
_binary_Navajo_size
_binary_Navajo_start
_binary_NZ_CHAT_end
_binary_NZ_CHAT_size
_binary_NZ_CHAT_start
_binary_NZ_end
_binary_NZ_size
_binary_NZ_start
_binary_Pacific_Apia_end
_binary_Pacific_Apia_size
_binary_Pacific_Apia_star
_binary_Pacific_Auckland_
_binary_Pacific_Auckland_
_binary_Pacific_Auckland_
_binary_Pacific_Bougainvi
_binary_Pacific_Bougainvi
_binary_Pacific_Bougainvi
_binary_Pacific_Chatham_e
_binary_Pacific_Chatham_s
_binary_Pacific_Chatham_s
_binary_Pacific_Chuuk_end
_binary_Pacific_Chuuk_siz
_binary_Pacific_Chuuk_sta
_binary_Pacific_Easter_en
_binary_Pacific_Easter_si
_binary_Pacific_Easter_st
_binary_Pacific_Efate_end
_binary_Pacific_Efate_siz
_binary_Pacific_Efate_sta
_binary_Pacific_Enderbury
_binary_Pacific_Enderbury
_binary_Pacific_Enderbury
_binary_Pacific_Fakaofo_e
_binary_Pacific_Fakaofo_s
_binary_Pacific_Fakaofo_s
_binary_Pacific_Fiji_end
_binary_Pacific_Fiji_size
_binary_Pacific_Fiji_star
_binary_Pacific_Funafuti_
_binary_Pacific_Funafuti_
_binary_Pacific_Funafuti_
_binary_Pacific_Galapagos
_binary_Pacific_Galapagos
_binary_Pacific_Galapagos
_binary_Pacific_Gambier_e
_binary_Pacific_Gambier_s
_binary_Pacific_Gambier_s
_binary_Pacific_Guadalcan
_binary_Pacific_Guadalcan
_binary_Pacific_Guadalcan
_binary_Pacific_Guam_end
_binary_Pacific_Guam_size
_binary_Pacific_Guam_star
_binary_Pacific_Honolulu_
_binary_Pacific_Honolulu_
_binary_Pacific_Honolulu_
_binary_Pacific_Johnston_
_binary_Pacific_Johnston_
_binary_Pacific_Johnston_
_binary_Pacific_Kiritimat
_binary_Pacific_Kiritimat
_binary_Pacific_Kiritimat
_binary_Pacific_Kosrae_en
_binary_Pacific_Kosrae_si
_binary_Pacific_Kosrae_st
_binary_Pacific_Kwajalein
_binary_Pacific_Kwajalein
_binary_Pacific_Kwajalein
_binary_Pacific_Majuro_en
_binary_Pacific_Majuro_si
_binary_Pacific_Majuro_st
_binary_Pacific_Marquesas
_binary_Pacific_Marquesas
_binary_Pacific_Marquesas
_binary_Pacific_Midway_en
_binary_Pacific_Midway_si
_binary_Pacific_Midway_st
_binary_Pacific_Nauru_end
_binary_Pacific_Nauru_siz
_binary_Pacific_Nauru_sta
_binary_Pacific_Niue_end
_binary_Pacific_Niue_size
_binary_Pacific_Niue_star
_binary_Pacific_Norfolk_e
_binary_Pacific_Norfolk_s
_binary_Pacific_Norfolk_s
_binary_Pacific_Noumea_en
_binary_Pacific_Noumea_si
_binary_Pacific_Noumea_st
_binary_Pacific_Pago_Pago
_binary_Pacific_Pago_Pago
_binary_Pacific_Pago_Pago
_binary_Pacific_Palau_end
_binary_Pacific_Palau_siz
_binary_Pacific_Palau_sta
_binary_Pacific_Pitcairn_
_binary_Pacific_Pitcairn_
_binary_Pacific_Pitcairn_
_binary_Pacific_Pohnpei_e
_binary_Pacific_Pohnpei_s
_binary_Pacific_Pohnpei_s
_binary_Pacific_Ponape_en
_binary_Pacific_Ponape_si
_binary_Pacific_Ponape_st
_binary_Pacific_Port_More
_binary_Pacific_Port_More
_binary_Pacific_Port_More
_binary_Pacific_Rarotonga
_binary_Pacific_Rarotonga
_binary_Pacific_Rarotonga
_binary_Pacific_Saipan_en
_binary_Pacific_Saipan_si
_binary_Pacific_Saipan_st
_binary_Pacific_Samoa_end
_binary_Pacific_Samoa_siz
_binary_Pacific_Samoa_sta
_binary_Pacific_Tahiti_en
_binary_Pacific_Tahiti_si
_binary_Pacific_Tahiti_st
_binary_Pacific_Tarawa_en
_binary_Pacific_Tarawa_si
_binary_Pacific_Tarawa_st
_binary_Pacific_Tongatapu
_binary_Pacific_Tongatapu
_binary_Pacific_Tongatapu
_binary_Pacific_Truk_end
_binary_Pacific_Truk_size
_binary_Pacific_Truk_star
_binary_Pacific_Wake_end
_binary_Pacific_Wake_size
_binary_Pacific_Wake_star
_binary_Pacific_Wallis_en
_binary_Pacific_Wallis_si
_binary_Pacific_Wallis_st
_binary_Pacific_Yap_end
_binary_Pacific_Yap_size
_binary_Pacific_Yap_start
_binary_Poland_end
_binary_Poland_size
_binary_Poland_start
_binary_Portugal_end
_binary_Portugal_size
_binary_Portugal_start
_binary_PRC_end
_binary_PRC_size
_binary_PRC_start
_binary_PST8PDT_end
_binary_PST8PDT_size
_binary_PST8PDT_start
_binary_ROC_end
_binary_ROC_size
_binary_ROC_start
_binary_ROK_end
_binary_ROK_size
_binary_ROK_start
_binary_Singapore_end
_binary_Singapore_size
_binary_Singapore_start
_binary_Turkey_end
_binary_Turkey_size
_binary_Turkey_start
_binary_UCT_end
_binary_UCT_size
_binary_UCT_start
_binary_Universal_end
_binary_Universal_size
_binary_Universal_start
_binary_US_Alaska_end
_binary_US_Alaska_size
_binary_US_Alaska_start
_binary_US_Aleutian_end
_binary_US_Aleutian_size
_binary_US_Aleutian_start
_binary_US_Arizona_end
_binary_US_Arizona_size
_binary_US_Arizona_start
_binary_US_Central_end
_binary_US_Central_size
_binary_US_Central_start
_binary_US_Eastern_end
_binary_US_Eastern_size
_binary_US_Eastern_start
_binary_US_East_Indiana_e
_binary_US_East_Indiana_s
_binary_US_East_Indiana_s
_binary_US_Hawaii_end
_binary_US_Hawaii_size
_binary_US_Hawaii_start
_binary_US_Indiana_Starke
_binary_US_Indiana_Starke
_binary_US_Indiana_Starke
_binary_US_Michigan_end
_binary_US_Michigan_size
_binary_US_Michigan_start
_binary_US_Mountain_end
_binary_US_Mountain_size
_binary_US_Mountain_start
_binary_US_Pacific_end
_binary_US_Pacific_size
_binary_US_Pacific_start
_binary_US_Samoa_end
_binary_US_Samoa_size
_binary_US_Samoa_start
_binary_UTC_end
_binary_UTC_size
_binary_UTC_start
_binary_WET_end
_binary_WET_size
_binary_WET_start
_binary_W_SU_end
_binary_W_SU_size
_binary_W_SU_start
_binary_Zulu_end
_binary_Zulu_size
_binary_Zulu_start
```

filimonov committed:
Trying to fix TZ with plus sign inside

filimonov committed:
one more attempt

objcopy creates symbols from source filenames (can't be ajusted), so just make temp files with needed names

filimonov commented on utils/ci/check-tzdata-updates.sh:
```diff
@@ -0,0 +1,18 @@
+else
+    echo "TZDB update required! Version in repo is ${tzdb_version_in_repo}, latest version is ${latest_tzdb_version}";
+    exit 1
+fi;
```
smth like that can be added to ci, to ensure we never miss tzdb updates (even if cctz maintainers will miss that)

alexey-milovidov commented:
test_materialize_mysql_database - already fixed.
01060_shutdown_table_after_detach - already fixed.
Stress test (address) - already fixed.

alexey-milovidov commented:
Looks like everything is perfect...
